environment/src/test/java/com/sequenceiq/environment/environment/dto/EnvironmentDtoConverterTest.java

```diff
@@ -0,0 +1,72 @@
+package com.sequenceiq.environment.environment.dto;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import com.sequenceiq.cloudbreak.auth.altus.EntitlementService;
+import com.sequenceiq.cloudbreak.auth.security.CrnUserDetailsService;
+import com.sequenceiq.cloudbreak.tag.CostTagging;
+import com.sequenceiq.environment.environment.domain.Environment;
+import com.sequenceiq.environment.environment.domain.EnvironmentAuthentication;
+import com.sequenceiq.environment.environment.validation.validators.PublicKeyValidator;
+import com.sequenceiq.environment.tags.service.AccountTagService;
+import com.sequenceiq.environment.tags.service.DefaultInternalAccountTagService;
+import com.sequenceiq.environment.tags.v1.converter.AccountTagToAccountTagResponsesConverter;
+
+class EnvironmentDtoConverterTest {
+
+    private final PublicKeyValidator publicKeyValidator = Mockito.mock(PublicKeyValidator.class);
+
+    private final CostTagging costTagging = Mockito.mock(CostTagging.class);
+
+    private final EntitlementService entitlementService = Mockito.mock(EntitlementService.class);
+
+    private final DefaultInternalAccountTagService defaultInternalAccountTagService = Mockito.mock(DefaultInternalAccountTagService.class);
+
+    private final AccountTagService accountTagService = Mockito.mock(AccountTagService.class);
+
+    private final CrnUserDetailsService crnUserDetailsService = Mockito.mock(CrnUserDetailsService.class);
+
+    private EnvironmentDtoConverter underTest = new EnvironmentDtoConverter(
+            new HashMap<>(),
+            new HashMap<>(),
+            new AuthenticationDtoConverter(publicKeyValidator),
+            costTagging,
+            entitlementService,
+            defaultInternalAccountTagService,
+            new AccountTagToAccountTagResponsesConverter(),
+            accountTagService,
+            crnUserDetailsService);
```

Comment:
Not a big deal but I suggest initializing this in a `@BeforeEach setUp()` method. Ideally converters are indeed stateless, but it is still a best practice to instantiate the test victim anew for each test method.